src/Widgets/EntryPopover/Generic.vala

```diff
@@ -27,31 +28,37 @@ public abstract class Tasks.Widgets.EntryPopover.Generic<T> : Gtk.EventBox {
     }
 
     construct {
-        events |= Gdk.EventMask.ENTER_NOTIFY_MASK
-            | Gdk.EventMask.LEAVE_NOTIFY_MASK;
+        popover = new Gtk.Popover (null);
+
+        var label = new Gtk.Label (placeholder);
 
-        popover = new Gtk.Popover (popover_button);
+        var popover_button_box = new Gtk.Box (HORIZONTAL, 0);
+        if (icon_name != null) {
+            popover_button_box.add (new Gtk.Image.from_icon_name (icon_name, BUTTON));
+        }
+        popover_button_box.add (label);
 
-        popover_button = new Gtk.MenuButton () {
-            label = placeholder,
+        var popover_button = new Gtk.MenuButton () {
+            child = popover_button_box,
             popover = popover,
-            image = new Gtk.Image.from_icon_name (icon_name, Gtk.IconSize.BUTTON),
             always_show_image = icon_name != null
```

Comment:
Do we still need to set `always_show_image` since now `image` property is not set?

Reply:
Oops thanks for catching that!